x-pack/plugins/endpoint/public/applications/endpoint/view/managing/index.tsx

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useMemo, useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentBody,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiTitle,
+  EuiBasicTable,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { createStructuredSelector } from 'reselect';
+import * as selectors from '../../store/endpoint_list/selectors';
+import { EndpointListAction } from '../../store/endpoint_list/action';
+import { useEndpointListSelector } from './hooks';
+import { usePageId } from '../use_page_id';
+import { CreateStructuredSelector } from '../../types';
+
+const selector = (createStructuredSelector as CreateStructuredSelector)(selectors);
+export const EndpointList = () => {
+  usePageId('managementPage');
+  const dispatch = useDispatch<(a: EndpointListAction) => void>();
+  const {
+    listData,
+    pageIndex,
+    pageSize,
+    totalHits: totalItemCount,
+    isLoading,
+  } = useEndpointListSelector(selector);
+
+  const paginationSetup = useMemo(() => {
+    return {
+      pageIndex,
+      pageSize,
+      totalItemCount,
+      pageSizeOptions: [10, 20, 50],
+      hidePerPageOptions: false,
+    };
+  }, [pageIndex, pageSize, totalItemCount]);
+
+  const onTableChange = useCallback(
+    ({ page }: { page: { index: number; size: number } }) => {
+      const { index, size } = page;
+      dispatch({
+        type: 'userPaginatedEndpointListTable',
+        payload: { pageIndex: index, pageSize: size },
+      });
+    },
+    [dispatch]
+  );
+
+  const columns = [
+    {
+      field: 'host.hostname',
+      name: i18n.translate('xpack.endpoint.management.list.host', {
+        defaultMessage: 'Hostname',
+      }),
+    },
+    {
+      field: '',
```

Comment:
Just noticed these - you have a few fields defined with no field name.

Reply:
is that ok? i guess i didn't need to change them, but those particular fields weren't mapped to the right place anyway since we the api doesn't return it yet

Reply:
I guess - probably need to create another "refactor" issue to remember to come back here and update all of these.
I guess I was surprised that this was working for the table - but I guess as long as you have a `render()` method, then its not an issue.

Reply:
Is there already another Issue tracking the need to come back here and refactor this when the API starts to return the real data? if so, add a comment and include a link 😃